test-suite/src/migrate.rs

```diff
@@ -33,7 +33,7 @@ test_case!(migrate, async move {
             "INSERT INTO Test (id, num, name) VALUES (1, 1, a.b);",
         ),
         (
-            EvaluateError::UnsupportedCompoundIdentifier(expr!("Here.User.id")).into(),
```

Comment:
As you removed `AggregateError::UnsupportedCompoundIdentifier`, we now no more need `EvaluateError::UnsupportedCompoundIdentifier`.